tribits/doc/developers_guide/TribitsDevelopersGuide.rst

```diff
@@ -6038,6 +6038,25 @@ Enable/Disable Logic`_).  Also, these files get processed in `Reduced Package
 Dependency Processing`_ as well so they get processed in all contexts where
 enable/disable logic is applied.
 
+How to tweak downstream TriBITS "ENABLE" variables during package configuration
+-------------------------------------------------------------------------------
+
+The internal configuration of a package (``<XXX>/CMakeLists.txt``)
+may determine that an optional feature (``<XXX>_ENABLE_<YYY>``) must be
+enabled or disabled with ``SET(<XXX>_ENABLE_<YYY> <val>)``.
+Changing the value of ``<XXX>_ENABLE_<YYY>`` inside package `<XXX>` using ``SET``
+changes the variable's value inside the package's scope, but all other
+packages will see the old value of ``<XXX>_ENABLE_<YYY>``.
+
+However, it is sometimes necessary to make this change visible to downstream packages.
+To do this, use ``DUAL_SCOPE_SET(<XXX>_ENABLE_<YYY> <val>)`` instead. This
+sets the value in both the global (project) scope and in the local scope of package `<XXX>`.
+Any downstream package (configured after `<XXX>` chronologically)
+will see the new value: ``${<XXX>_ENABLE_<YYY>} STREQUAL <val>``.
+Caution: packages that don't depend on `<XXX>` may or may not see the change.
```

Comment:
@brian-kelley, what is the purpose of:

> Caution: packages that don't depend on `<XXX>` may or may not see the change.

?

Under what situations would an upstream package change some variable `<XXX>_ENABLE_<YYY>` and a downstream package's `<packageDir>/CMakeLists.txt` file would not see it?  If if it did not see it, why do we care (because one would assume it was not dependent on that variable).

Reply:
@bartlettroscoe What I meant by this is that the change won't be visible to any non-downstream package. But I agree that a) we don't care and b) this is kind of reiterating the last sentence in a more confusing way.